wa_examples/html_wa.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "/wa/translation-alignment/QWxpZ25tZW50Q2h1bmtOb2RlOjE=",
+  "type": "Annotation",
+  "target": [
+    "urn:cts:greekLit:tlg0012.tlg001.msA:1.1-1.7",
+    {
+      "type": "SpecificResource",
+      "source": {
+        "id": "https://rosetest.library.jhu.edu/rosademo/iiif/homer/VA/VA012RN-0013/canvas",
+        "type": "Canvas"
+      },
+      "selector": {
+        "type": "ImageApiSelector",
+        "region": "pct:6.11,22.52,46.75,15.01"
```

Comment:
https://image.library.jhu.edu/iiif/homer%2FVA%2FVA012RN-0013/pct:6.11,22.52,46.75,15.01/full/0/default.jpg

Reply:
I see this is using ImageApiSelector from https://iiif.io/api/annex/openannotation/
I'm not sure it can be used with a Canvas. I think its meant to be used directly with IIIF Image URI.

This starts to beg the question of whether to use IIIF 2 or 3. We've been playing it a little loose. The examples I gave in the last doc used IIIF 3 and the recommended FragmentSelector with Canvas. For the moment I think we should just continue with IIIF 3 with /iiif3. (I may try to join the IIIF 2 and 3 endpoints using content negotiation.)